### PR TITLE
12 general css homepage issues

### PR DESCRIPTION
**Pull request by LakGillJPN**
Fixed various CSS issues on the homepage to make it more presentable.

### CODE REVIEW

File: client/src/components/Card.css [hunk 1]
```diff
@@ -1,27 +1,88 @@
...
+
+.detailsButton {
+ background-color: #66BFBF;
+ height: 50px;
+ width: 100px;
+ border-radius: 10px;
+ font-size: 15px;
+ color: white;
+ border-color: 1px solid #66a7bf
 }
```
Comment (kappanjoe) [inline client/src/components/Card.css:43]: nit: it appears the indent is not consistent here with the rest of the file

File: client/src/components/Card.jsx [hunk 2]
```diff
@@ -10,19 +10,36 @@ const Card = (props) => {
...
+   if (averageScore > 7) {
+    setBackground('green')
+   }
+   else if (averageScore > 5) {
+    setBackground('yellow')
+   }
+   else {
+    setBackground('red')
+   }
+  },[averageScore])
```
Comment (kappanjoe) [inline client/src/components/Card.jsx:26]: nit: indentation appears to only be one space - the rest of the document uses two spaces

**Review by kappanjoe — COMMENTED** [review]
LGTM! Only thing I noticed was minor code styling issues but the code itself looks great.

File: client/src/components/Card.css (hunk 1 above)
Comment (LakGillJPN) [reply client/src/components/Card.css]: thanks, fixed it!

**Pull request merged**